lib/controllers/v1/places_controller.js

```diff
@@ -199,22 +202,26 @@ PlacesController.show = async req => {
       }
     ];
   }
+  const body = {
+    sort,
+    query: {
+      bool: {
+        should: [
+          { terms: { id: numericIDs } },
+          { terms: { slug: ids } }
+        ]
+      }
+    },
```

Comment:
I guess I forgot to submit this commit earlier. The way this query is formed you'd need to add `minimum_should_match: 1` to the bool object to do what is intended here - force at least one of the should clauses to be true. As the documentation at https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-bool-query.html#bool-min-should-match says:

```
If the bool query includes at least one should clause and no must or filter clauses, the default value is 1. Otherwise, the default value is 0.
```

And in my testing if you provide an admin_level query param, that's the only filter applied and it ignores the ID

Reply:
Thank you! Sorry for not covering it with my testing. Fixed and added a test for this case